### PR TITLE
[JS] Topic: jQuery

### LINKED ISSUE #2882: [Overview of Popular libraries: jQuery]:
### Description

[Path name](https://github.com/girlscript/winter-of-contributing/tree/Javascript/Web_Development/Javascript/Topics)

Query is a lightweight, "write less, do more", JavaScript library. The purpose of jQuery is to make it much easier to use JavaScript on your website. jQuery also simplifies a lot of the complicated things from JavaScript, like AJAX calls and DOM manipulation. The jQuery library contains the following features:

- HTML/DOM manipulation
- CSS manipulation
- HTML event methods
- Effects and animations
- AJAX
- Utilities

I am a GWOC'21 contributor from Batch-1 of Javascript. Please assign this issue to me. I assure you that this issue's title will be the same as my PR's title as well as I will be Contributing under the Guidelines and Code of Conduct.

### Domain

JavaScript

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by Aditi-247**
<hr>

## Description 📜
I made a documentation on JS:jQuery that includes few points like:-

--> What is jQuery?
--> What are the uses of jQuery?
--> What is jQuery History?
--> What are the properies of jQuery?
--> What is the difference between JavaScript and jQuery?
--> What are the effects methods used in jQuery?
Fixes #2882

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [x] JavaScript
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by s4-2** [comment]
@Aditi-247 Please put correct format as #2882 in Description.

File: JavaScript/Topics/17.jQuery/README.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  --> Netflix<br>
+
+# What should you know before starting to learn jQuery?
+  It is always advised to a fresher to learn the basics of web designing before starting to learn jQuery. He should learn HTML, CSS and JavaScript first. But, if you belong to a technical background, it is up to you.
+
+# what is jQuery History?
+  jQuery was first released in January 2006 by John Resig at BarCamp NYC. It is currently headed by Timmy Wilson and maintained by a team of developers. Nowadays, jQuery is widely used technology. Most of the websites are using jQuery.
+
+  jQuery Release History-
+
```
Comment (s4-2) [inline JavaScript/Topics/17.jQuery/README.md:42]: Just provide first release version of jQuery, the documentation should be precise:)

File: JavaScript/Topics/17.jQuery/README.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+# Is jQuery library used for server scripting or client scripting?
+  It is a library for client-side Scripting.
+
+# Is jQuery a W3C standard?
+  No, jQuery is not a W3C standard.
+
+# How to add jQuery to your web pages?
+  There are several ways to start using jQuery on your web site. You can:<br>
+
+  --> Download the jQuery library from jQuery.com<br>
```
Comment (s4-2) [inline JavaScript/Topics/17.jQuery/README.md:69]: Instead of **-->** , use **use bullet points**

File: JavaScript/Topics/17.jQuery/README.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  <head><br>
+    <script src="jquery-3.5.1.min.js"></script><br>
+  </head><br>
+  Important Points: Place the downloaded file in the same directory as the pages where you wish to use it.<br>
+
+  --> jQuery CDN: If you don't want to download and host jQuery yourself, you can include it from a  CDN (Content Delivery Network).<br>
+  Google is an example of someone who host jQuery:<br>
+  Google CDN:<br>
+  <head><br>
+     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script><br>
```
Comment (s4-2) [inline JavaScript/Topics/17.jQuery/README.md:89]: Separate the </script> tag as it is also coming under the link.

File: JavaScript/Topics/17.jQuery/README.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+  --> fadeOut() - It shows the matched elements by fading it to transparent. In other words, it fades out the selected elements.<br>
+
+# jQuery Properties are:
+
+--> context: Removed in version 3.0. Contains the original context passed to jQuery()<br>
+--> jquery:  Contains the jQuery version number<br>
+--> jQuery.fx.interval: Change the animation firing rate in milliseconds<br>
+--> jQuery.fx.off: Globally disable/enable all animations<br>
+--> jQuery.support: A collection of properties representing different browser features  or bugs (Intended for jQuery's internal use)<br>
+--> length:	Contains the number of elements in the jQuery object<br>
```
Comment (s4-2) [inline JavaScript/Topics/17.jQuery/README.md:138]: At last, mention your name as contributor **__Contributor__: XYZ** like this

**Comment by Aditi-247** [comment]
@paramsiddharth sir I made all the changes that mam suggested. can u please check it sir

**Comment by Aditi-247** [comment]
@s4-2 And mam please can you please review it .

File: JavaScript/Topics/17.jQuery/README.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+<ul>
+  <li> context: Removed in version 3.0. Contains the original context passed to jQuery()</li>
+ <li> jquery:  Contains the jQuery version number</li>
+ <li> jQuery.fx.interval: Change the animation firing rate in milliseconds</li>
+ <li> jQuery.fx.off: Globally disable/enable all animations</li>
+ <li> jQuery.support: A collection of properties representing different browser features  or bugs (Intended for jQuery's internal use)</li>
+ <li> length:	Contains the number of elements in the jQuery object</li>    
+  </ul>
+  
+ Contributor : Aditi Tomar
```
Comment (s4-2) [inline JavaScript/Topics/17.jQuery/README.md:134]: Please add like this 
__Contributor__ : [Your name](your github link)

**Comment by Aditi-247** [comment]
@s4-2 @paramsiddharth I added all the changes . If any other so please suggest . Please review it or please merge . Because it's my 1st contribution

**Comment by Aditi-247** [comment]
@s4-2 Ma'am . I made changes . please check it mam

File: JavaScript/Topics/17.jQuery/README.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+ </ul> 
+  There are two versions of jQuery available for downloading:<br>
+<ul>
+  <li> Production version - this is for your live website because it has been minified and compressed</li>
+  <li> Development version - this is for testing and development (uncompressed and readable code)</li>
+  <li> Both versions can be downloaded from jQuery.com.</li>
+</ul>
+The jQuery library is a single JavaScript file, and you reference it with the HTML <script> tag (note that the <script> tag should be inside the <head> section):<br>
+ Syntax: <head>
+            <script src="jquery-3.5.1.min.js"></script><br>
```
Comment (s4-2) [inline JavaScript/Topics/17.jQuery/README.md:69]: See the image, </script> should not come under the link. Please correct it:)

**Comment by s4-2** [comment]
IMG_20210930_164740

**Comment by Aditi-247** [comment]
@s4-2 Ma'am I didn't understand what I have to change. Acc. to image I changed . please check it

**Review by paramsiddharth — APPROVED** [review]
@Creatoon @dakshsethi @s4-2 Please review.

**Comment by s4-2** [comment]
> @s4-2 Ma'am I didn't understand what I have to change. Acc. to image I changed . please check it

Wow @Aditi-247 great job:-)

**Review by s4-2 — APPROVED** [review]

File: JavaScript/Topics/17.jQuery/README.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
+# What is jQuery ?
+
+  Query is a fast, small, cross-platform and feature-rich JavaScript library. It is designed to simplify the client-side scripting of HTML.jQuery is a lightweight, "write     less, do more", JavaScript library.jQuery takes a lot of common tasks that require many lines of JavaScript code to accomplish, and wraps them into methods that you can call with a single line of code.jQuery also simplifies a lot of the complicated things from JavaScript, like AJAX calls and DOM manipulation.
```
Comment (dakshsethi) [inline JavaScript/Topics/17.jQuery/README.md:3]:
```suggestion
Query is a fast, small, cross-platform and feature-rich JavaScript library. It is designed to simplify the client-side scripting of HTML. **jQuery is a lightweight, "write less, do more", JavaScript library**. jQuery takes a lot of common tasks that require many lines of JavaScript code to accomplish, and wraps them into methods that you can call with a single line of code.jQuery also simplifies a lot of the complicated things from JavaScript, like AJAX calls and DOM manipulation.
```

File: JavaScript/Topics/17.jQuery/README.md [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+  <ul>
+  <li> HTML/DOM manipulation</li>
+  <li> CSS manipulation</li>
+  <li> HTML event methods</li>
+  <li> Effects and animations</li>
+  <li> AJAX</li>
+  <li> Utilities</li>
+  </ul>
+
+# Is jQuery a programming language?
```
Comment (dakshsethi) [inline JavaScript/Topics/17.jQuery/README.md:17]:
```suggestion
## Is jQuery a programming language?
```

File: JavaScript/Topics/17.jQuery/README.md [hunk 9]
```diff
@@ -0,0 +1,131 @@
...
+  jQuery is not a programming language but a well-written JavaScript code. It is used to traverse documents, event handling, Ajax interaction, and Animation.
+
+# What is the difference between JavaScript and jQuery?
```
Comment (dakshsethi) [inline JavaScript/Topics/17.jQuery/README.md:20]:
```suggestion
## What is the difference between JavaScript and jQuery?
```

File: JavaScript/Topics/17.jQuery/README.md [hunk 10]
```diff
@@ -0,0 +1,131 @@
...
+  The simple difference is that JavaScript is a language while jQuery is a built-in library built for JavaScript. jQuery simplifies the use of JavaScript language.  
+
+# Why jQuery is required ?
```
Comment (dakshsethi) [inline JavaScript/Topics/17.jQuery/README.md:23]:
```suggestion
## Why jQuery is required ?
```

File: JavaScript/Topics/17.jQuery/README.md [hunk 11]
```diff
@@ -0,0 +1,131 @@
...
+
+  Some of these companies are:<br>
+  <ul>
+    <li> Microsoft</li>
+    <li> Google</li>
+    <li> IBM</li>
+    <li> Netflix</li>
+  </ul>
+
+# What should you know before starting to learn jQuery?
```
Comment (dakshsethi) [inline JavaScript/Topics/17.jQuery/README.md:42]: Change all of `#` to `##`

**Review by dakshsethi — CHANGES_REQUESTED** [review]

File: JavaScript/Topics/17.jQuery/README.md (hunk 11 above)
Comment (Aditi-247) [reply JavaScript/Topics/17.jQuery/README.md]: @dakshsethi Sir I add ## in documentation . Any other changes

**Comment by Aditi-247** [comment]
@dakshsethi @paramsiddharth sir please review if any other changes are required so please tell me and if not so please merge it sir .

**Comment by paramsiddharth** [comment]
@dakshsethi Please review.

**Comment by Aditi-247** [comment]
@s4-2 Thankyou mam @paramsiddharth Thankyou sir @dakshsethi Sir please review .When this merge then i will create new issue sir . Please sir

**Review by dakshsethi — APPROVED** [review]
Looks good to me!! 🔥🔥🔥

**Comment by Aditi-247** [comment]
@paramsiddharth @dakshsethi @s4-2  sir/ma'am is my PR get merged or not . Shall I close it or not

**Comment by dakshsethi** [comment]
> @paramsiddharth @dakshsethi @s4-2 sir/ma'am is my PR get merged or not . Shall I close it or not

It will get merged since it is approved by all of us.

**Comment by Aditi-247** [comment]
@dakshsethi  Means sir my one contribution is successful. It will merged right

**Comment by Aditi-247** [comment]
@paramsiddharth yes sir. It is ready please merge   it

**Comment by Aditi-247** [comment]
@paramsiddharth Sir when will this document merged

**Comment by dakshsethi** [comment]
> @paramsiddharth Sir when will this document merged

Please wait for some time, it is possible that @paramsiddharth must be occupied with other things, just leave this PR here for some time as it is approved by all of us. So it will 100% get merged!!

**Comment by Aditi-247** [comment]
@dakshsethi @paramsiddharth ok sir then I will start contributing on my next issue

**Comment by paramsiddharth** [comment]
@Aditi-247 @dakshsethi I'm sorry I took too long to merge it, but now's the time! 💕 Good job!

**Comment by Aditi-247** [comment]
@paramsiddharth Thanku sir

**Pull request merged**